pyproject.toml

```diff
@@ -1,3 +1,112 @@
+# Package ######################################################################
+
+[build-system]
+requires = ["setuptools >= 61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "gymnasium"
+description = "A standard API for reinforcement learning and a diverse set of reference environments (formerly Gym)."
+readme = "README.md"
+requires-python = ">= 3.7"
+authors = [{ name = "Farama Foundation", email = "contact@farama.org" }]
+license = { text = "MIT License" }
+keywords = ["Reinforcement Learning", "game", "RL", "AI", "gymnasium"]
+classifiers = [
+    "Development Status :: 4 - Beta",  # change to `5 - Production/Stable` when ready
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+dependencies = [
+    "numpy >=1.21.0",
+    "jax-jumpy >=0.2.0",
+    "cloudpickle >=1.2.0",
+    "importlib_metadata >=4.8.0; python_version < '3.10'",
+    "gymnasium_notices >=0.0.1",
+    "shimmy >=0.1.0,<1.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+# Update dependencies in `all` if any are added or removed
+atari = ["shimmy[atari] >=0.1.0,<1.0"]
+accept-rom-license = ["autorom[accept-rom-license] ~=0.4.2"]
+box2d = ["box2d-py ==2.3.5", "pygame ==2.1.0", "swig ==4.*"]
+classic-control = ["pygame ==2.1.0"]
+classic_control = ["pygame ==2.1.0"]  # kept for backward compatibility
+mujoco-py = ["mujoco-py >=2.1,<2.2"]
+mujoco_py = ["mujoco-py >=2.1,<2.2"]  # kept for backward compatibility
+mujoco = ["mujoco >=2.3.0", "imageio >=2.14.1"]
```

Comment:
The extra-require name is also lower-cased to fit the [PEP 503: Normalized Names](https://peps.python.org/pep-0503/#normalized-names) convention as commented above. The old names are kept for backward compatibility:

```bash
pip3 install gymnasium[classic-control]
pip3 install gymnasium[classic_control]
```